src/schema/__tests__/artists.test.js

```diff
@@ -26,4 +26,27 @@ describe("Artists", () => {
     const { artists } = await runQuery(query, { artistsLoader })
     expect(artists).toEqual([{ name: "Han Myung-Ok" }])
   })
+
+  it("returns a list of artists matching array of ids", async () => {
+    const artistsLoader = ({ ids }) => {
+      if (ids) {
+        return Promise.resolve([
+          {
+            _id: "52c721e5b202a3edf1000072",
+            name: "Han Myung-Ok",
+          },
+        ])
+      }
+      throw new Error("Unexpected invocation")
+    }
```

Comment:
I was actually thinking of _using_ the passed in `ids`, so something like:

```js
const artistsLoader = ({ ids }) => Promise.resolve(ids.map(_id => ({ _id })))
```

That way you can rest assured that the data you put in influences the data that comes out.

Reply:
Oh I see yes-- thanks for clarifying this test pattern

Reply:
This is updated now, do you have other changes before this is ready to merge?